libs/s3/s3_test_lib.py

```diff
@@ -640,19 +640,24 @@ def get_object(
             bucket: str = None,
             key: str = None,
             ranges: str = None,
-            raise_exec: bool = True) -> tuple:
+            raise_exec: bool = True,
```

Comment:
Better to keep this optional parameters in kwargs.

Reply:
done